components/inputs/input-checkbox.js

```diff
@@ -62,6 +63,10 @@ class InputCheckbox extends RtlMixin(LitElement) {
 				.d2l-input-checkbox-text-disabled {
 					opacity: 0.5;
 				}
+				input.d2l-input-checkbox.align-top {
+					vertical-align: top;
+					margin-top: 0.2em;
```

Comment:
`0.2em` doesn't seem like the right value here. If I overlay the text baseline, there's more of the checkbox above the text than below.

![Screen Shot 2020-04-15 at 12 59 44 PM](https://user-images.githubusercontent.com/5491151/79366214-cd177d00-7f19-11ea-9e2d-10d034d301f3.png)

I'd defer to a designer (@geurts any thoughts?), but the correct value to me seems to be `0`:

![Screen Shot 2020-04-15 at 1 04 21 PM](https://user-images.githubusercontent.com/5491151/79366237-d274c780-7f19-11ea-9bfe-f4d2015aa048.png)

Reply:
Using `0` makes it way too high:
![image](https://user-images.githubusercontent.com/12971370/79366944-02709a80-7f1b-11ea-8e97-e4de77720dfc.png)

Reply:
Actually, I think changing the alignment of the text label might fix it without the need for a margin

Reply:
Oh, I realized what's wrong. I'm setting the styling on the wrong element.

Reply:
Hmm so that to me indicates that there's other factors at play here. If I apply `align-top` to a single-line checkbox I end up with something similar where the alignment seems "off":

![Screen Shot 2020-04-15 at 1 23 47 PM](https://user-images.githubusercontent.com/5491151/79367765-66e02980-7f1c-11ea-90a5-4931b1c94843.png)

Have you considered just using a native checkbox element and [applying our checkbox styles](https://github.com/BrightspaceUI/core/tree/master/components/inputs#applying-styles-to-native-checkboxes) to it? Then you could do whatever you like to it. I still think toying with this kind of top alignment is possibly still desire-able, just want to make sure we're doing it correctly to handle all cases.

Reply:
With my latest changes, it now seems to be correctly centred:
![image](https://user-images.githubusercontent.com/12971370/79368204-056c8a80-7f1d-11ea-8d0f-d1686a8b6136.png)

![image](https://user-images.githubusercontent.com/12971370/79368288-259c4980-7f1d-11ea-8a3f-3dcf4029b9aa.png)